packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -1358,6 +1372,9 @@ Adds an OAuth 2.0 client application. This application is only available to the
 | backchannel_authentication_request_signing_alg <ApiLifecycle access="ie" /> | The signing algorithm for Client-Initiated Backchannel Authentication signed requests. If this value isn't set and a signed request is sent, the request fails.                                                                     | String                                                                                        | TRUE       | FALSE    | TRUE      |
 | backchannel_custom_authenticator_id <ApiLifecycle access="ie" />             | The ID of the custom authenticator that authenticates the user.                                                                                                                                                        | String                                                                                        | TRUE       | FALSE    | TRUE      |
 | dpop_bound_access_tokens  <ApiLifecycle access="ea" />                                                    | Indicates that the client application uses Demonstrating Proof-of-Possession (DPoP) for token requests. If omitted, the default value is `false`. If `true`, the authorization server rejects token requests from this client that don't contain the DPoP header.  | Boolean  | TRUE       | FALSE    | TRUE      |
+| participate_slo <ApiLifecycle access="ie" /> <ApiLifecycle access="ea" />   | Determines if the application participates in Single Logout                      | Boolean  | TRUE       | FALSE    | TRUE      |
+| frontchannel_logout_uri <ApiLifecycle access="ie" /> <ApiLifecycle access="ea" />   | URL where Okta sends the logout request                            | Boolean  | TRUE       | FALSE    | TRUE      |
+| frontchannel_logout_session_required <ApiLifecycle access="ie" /> <ApiLifecycle access="ea" />   | Session details allow you to end a specific user session instead of all active user sessions    | Boolean  | TRUE       | FALSE    | TRUE      |
```

Comment:
```suggestion
| frontchannel_logout_session_required <ApiLifecycle access="ie" /> <ApiLifecycle access="ea" />   | Allows you to end a specific user session instead of all active user sessions    | Boolean  | TRUE       | FALSE    | TRUE      |
```